app/assets/javascripts/app/components/comments_list/comments_list.js

```diff
@@ -27,7 +27,7 @@
     ctrl.cancelEdit = cancelEdit;
 
     function updateComment(comment) {
-      var updated = DS.createInstance('discussionComment', comment, { useClass:true });
+      var updated = DS.createInstance('comment', comment, { useClass:true });
```

Comment:
I've added a trello card for this
On Fri 16 Jan 2015 at 21:41 Bruno Azevedo <notifications@github.com> wrote:

> In app/assets/javascripts/app/components/comments_list/comments_list.js
> <https://github.com/groupbuddies/council/pull/94#discussion-diff-23111491>
> :
>
> > @@ -27,7 +27,7 @@
> >      ctrl.cancelEdit = cancelEdit;
> >
> >      function updateComment(comment) {
> > -      var updated = DS.createInstance('discussionComment', comment, { useClass:true });
> > +      var updated = DS.createInstance('comment', comment, { useClass:true });
>
> I'd really like these random instance creations from an object that's
> being pass around in the views not to happen. Couldn't the comment passed
> in already be a DS instance?
> We've already discussed this more thoroughly in person.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/groupbuddies/council/pull/94/files#r23111491>.
>